packages/docs/stories/components/tooltip.stories.ts

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import '../../../components/src/components/tooltip/tooltip.js';
+import {
+  generateScreenshotStory,
+  generateStoryDescription,
+  storybookDefaults,
+  storybookHelpers,
+  storybookTemplate,
+} from '../../src/helpers/component.js';
+import { generateFigmaPluginObject } from '../../src/helpers/figma.js';
+
+const { args: defaultArgs, argTypes } = storybookDefaults('syn-tooltip');
+const { overrideArgs } = storybookHelpers('syn-tooltip');
+const { generateTemplate } = storybookTemplate('syn-tooltip');
+
+const meta: Meta = {
+  args: overrideArgs([
+    {
+      name: 'content',
+      type: 'attribute',
+      value: 'This is a tooltip',
+    },
+    {
+      name: 'open',
+      type: 'attribute',
+      value: true,
+    },
+    {
+      name: 'default',
+      type: 'slot',
+      value: '<syn-button>Hover me</syn-button>',
+    },
+  ], defaultArgs),
+  argTypes,
+  component: 'syn-tooltip',
+  parameters: {
+    design: generateFigmaPluginObject('12852-79550'),
+    docs: {
+      description: {
+        component: generateStoryDescription('tooltip', 'default'),
+      },
+      story: {
+        height: '200px',
+        // Unfortunately we need the iframes here, otherwise the tooltips placing is broken
+        inline: false,
+      },
+    },
+    layout: 'centered',
+  },
+  title: 'Components/syn-tooltip',
+};
+export default meta;
+
+type Story = StoryObj;
+
+export const Default: Story = {
+  parameters: {
+    controls: {
+      disable: false,
+    },
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'default'),
+      },
+    },
+  },
+  render: (args: unknown) => generateTemplate({ args }),
+} as Story;
+
+export const Placement: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('tooltip', 'placement'),
+      },
+      story: {
+        height: '450px',
+      },
+    },
+    layout: 'padded',
+  },
+  render: () => html`
+    <div class="tooltip-placement-example">
+      <div class="tooltip-placement-example-row">
+        <syn-tooltip content="top-start" placement="top-start" open>
+          <syn-button></syn-button>
+        </syn-tooltip>
+
+        <syn-tooltip content="top" placement="top" open>
+          <syn-button></syn-button>
+        </syn-tooltip>
+
+        <syn-tooltip content="top-end" placement="top-end" open>
+          <syn-button></syn-button>
+        </syn-tooltip>
+      </div>
+
+      <div class="tooltip-placement-example-row">
+        <syn-tooltip content="left-start" placement="left-start" open>
+          <syn-button></syn-button>
```

Comment:
When adding trigger="manual", the items don´t close when hovering. I think this would be a fine addition for viewing in Storybook, what do you think?


```suggestion
        <syn-tooltip content="left-start" placement="left-start" trigger="manual" open>
```

Reply:
What you mean is  trigger="click"  so it will just close and open on click.  I think its okay like this. But I don`t mind if we change the behavior from `hover` to `click`

Reply:
Then I am fine, too :)